src/Common/Config/Kremlin.cpp

```diff
@@ -152,6 +163,19 @@ bool Kremlin::CreateGenericCommandGroups(bool lastTry) {
 			if (missingCommand) {
 				continue;
 			}
+			YAML::Node defaultConfig;
+			if (defaultConfigs.count(fullName) == 0) {
+				if (lastTry) {
+					defaultConfig = YAML::Node();
+				} else {
+					continue;
+				}
+			} else {
+				defaultConfig = defaultConfigs[fullName];
+			}
+			for (YAML::Node subnode : defaultConfig) {
+				commandConfig.push_back(subnode);
+			}
```

Comment:
I think this is only executed for CommandGroup, and those generally don't need defaults. I think Zach and I were thinking more of something for regular commands.